docs/servers/ssh-into-server.md

```diff
@@ -7,7 +7,7 @@ links:
     previous:
         - /docs/servers
     next:
-        - /docs/applications/add-application
+        - /docs/servers/check-server-provision-logs
```

Comment:
Next step of `SSH into server` is checking provision logs? 🤔 
Shouldn't user check provision logs right after provision a server, thus a way before SSH-ing into it?

Reply:
Will fix; I believe the best order for this post would be after adding a server and before ssh into it.

Reply:
Updated links for the introduction guide, now the current reading flow is:

```mermaid
flowchart TD
   A[add-aws-server] --> D[check-server-provision-logs]
   B[add-existing-server] --> D
   C[add-gcp-server] --> D
   D --> E[ssh-into-server]
```